samples/getting-started/DemoApp/ViewController.swift

```diff
@@ -15,12 +15,31 @@ class ViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        label.text = "Hello World"
-        label.font = .systemFont(ofSize: 30)
-        label.sizeToFit()
-        label.center = view.center
+        let scrollView = UIScrollView(frame: CGRect(x: 200, y: 100, width: 300, height: 150))
+        scrollView.contentSize = CGSize(width: 600, height: 300)
+        scrollView.backgroundColor = .white
+
+        let colors: [UIColor] = [.green, .orange, .red]
+        var views = [UIView]()
+
+        for (i, color) in colors.enumerated() {
+            let view = UIView(frame: CGRect(origin: CGPoint(x: 220+20*i, y: 10+20*i), size: CGSize(width: 100, height: 100)))
+            view.backgroundColor = color
+            views.append(view)
+            scrollView.addSubview(view)
+        }
+
+
+        let blueView = UIView(frame: CGRect(origin: CGPoint(x: 225+20*2 - 50, y: 15+20*2), size: CGSize(width: 150, height: 10)))
```

Comment:
please always use a space between binary operators
also, this line is very long, it would be great if you could split it up logically into

`let blueViewFrame = CGRect(x:,y:,width:,height:)` (avoiding the origin:,size: initializer)

also, these values seem pretty arbitrary, would be good if you could put them into named variables so they get some meaning

Reply:
(pasting in both relevant places) 
Very sorry about this, the changes to this file were not meant to be submitted. They were useful to me while developing, but my goal wasn't to change the DemoApp. I was pushing it to keep the git index clean, with the intention of resetting the file before re-requesting review.